modules/ocf_kubernetes/manifests/master.pp

```diff
@@ -20,21 +20,66 @@
   $etcd_archive = "etcd-v${etcd_version}-linux-amd64.tar.gz"
   $etcd_source  = "https://github.com/etcd-io/etcd/releases/download/v${etcd_version}/${etcd_archive}"
 
+  class { 'ocf_kubernetes::package::first_stage':
+    stage => first,
+  }
+
+  # Passwords for the static token file
+  # https://kubernetes.io/docs/reference/access-authn-authz/authentication/#static-token-file
+  $ocf_jenkins_deploy_token = lookup('kubernetes::jenkins_token')
+
+  file {
+    '/etc/kubernetes-ocf':
+      ensure  => directory,
+      recurse => true,
+      purge   => true;
+
+    '/etc/kubernetes-ocf/static-tokens.csv':
+      content   => template('ocf_kubernetes/static-tokens.csv.erb'),
```

Comment:
Instead of making this a whole template, you could just make it an inline content block (like https://github.com/ocf/puppet/blob/34d1a1fe8f850dac0f008b7521d8dbaedddf09d3/modules/ocf_mesos/manifests/slave.pp#L15 or something) since it's just a line. Either way is fine, but it might just be a bit cleaner to avoid having a whole separate file for a single line of content.

I'd say this depends on if you think you'd need to add multiple lines to that file or how complicated that file is likely to get.

Reply:
I'd like to leave open the possibility of adding more to this file.